src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/index.jelly

```diff
@@ -41,36 +41,42 @@
 							<td class="pane scriptler-toolbar" width="1%">
 								<j:choose>
 									<j:when test="${approved == null}">
-										<img width="16" height="16" title="${%fileNotAvailable}" src="${imagesURL}/16x16/red.gif" class="scriptler-icon-slot-first" />
+										<a class="scriptler-icon-slot">
+											<l:icon title="${%fileNotAvailable}" class="icon-error icon-sm" />
+										</a>
```

Comment:
Please remove the `<a>` element around this icon to be the same as the approval and warning icons. Adding `class="scriptler-icon-slot"` here will result in a `margin-left` of 8px being applied to this element, where it was specifically 0px before.